SimExLite/WavefrontData/WPGFormat.py

```diff
@@ -0,0 +1,126 @@
+import numpy as np
+import h5py
+from libpyvinyl.BaseFormat import BaseFormat
+from .WavefrontData import WavefrontData
```

Comment:
i like it when local imports and system lib imports are separated by at least one line